tests/loggers/test_mlflow_logger.py

```diff
@@ -22,6 +22,16 @@
 from tests.models.test_hf_model import check_hf_model_equivalence, check_hf_tokenizer_equivalence
 
 
+@pytest.fixture(autouse=True)
```

Comment:
does this have to be autouse=True? we are moving away from autouse=True because it makes debugging really hard. @mvpatel2000 Can you just inject it into the other MLflow tests?

Reply:
This should only take effect in this test, but I can inline the code if this is against mosaic style. Let me know which one you prefer, thanks!